Configure

```diff
@@ -541,7 +542,7 @@ my @disablables = (
     "thread-pool",
     "threads",
     "tls",
-    "integrity-only-ciphers",
```

Comment:
Did you intentionally remove 'integrity-only-ciphers" ?

Reply:
It has an entry in INSTALL.md

Reply:
It got **moved** not *removed*.